### ISSUE TITLE
508-Defect-3 [AXE-CORE]: Headings MUST not be empty

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#12672]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

**Feedback framework**
- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

On the review screen there is `<h3 class="form-review-panel-page-header"> </h3>
`, which is an empty h3.

### Why it Matters
Screen readers alert users to the presence of a heading tag. If the heading is empty or the text cannot be accessed, this could either confuse users or even prevent them from accessing information on the page's structure.

If the text inside a heading cannot be accessed by a screen reader, users of this technology will not be able to hear the contents of the heading. Since headings relay the structure of a webpage, it's crucial that users of screen readers are able to access the contents.

Applying heading markup (`<h1>` through `<h6>`) is a quick way to make text stand out, however, using it for anything other than headings will make navigating a web page more confusing for users of assistive technology.

In the same way that sighted users can glance at a page and get a sense of its contents, users of screen readers can do the same by navigating through headings. Well written and properly ordered headings can save users, especially those who use screen readers, a lot of time and frustration.

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a screen reader user, I want to navigate the content of the page and understand the content structure by heading levels.

## Environment

* Operating System: all
* Browser: any
* Screenreading device: any
* Server destination: staging

## Steps to Recreate

1. Logged in as user 261, enter https://staging.va.gov/disability/file-disability-claim-form-21-526ez/review-and-submit in browser
2. This should be on the Review screen
3. Using developer tools, run an axe scan
4. Verify that the "Headings must not be empty" appears, related to:

```
Element location
.form-review-panel-page:nth-child(25) > .form-review-array > div > .form-review-panel-page-header-row > .form-review-panel-page-header
Element source
<h3 class="form-review-panel-page-header"> </h3>
```

## Possible Fixes (optional)

```diff
! <h3 class="form-review-panel-page-header"> </h3>
+ <h3 class="form-review-panel-page-header"> Time Period </h3>
```

## WCAG or Vendor Guidance (optional)

* [Deque University - Headings must not be empty, best practice](https://dequeuniversity.com/rules/axe/3.5/empty-heading)

## Screenshots or Trace Logs

![Screen Shot 2020-08-20 at 1 36 42 PM](https://user-images.githubusercontent.com/57469/90805955-57b62180-e2ea-11ea-9dab-f3e6cd2e4f48.png)

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

**Comment by Mottie** [comment]
This fix should now be available in staging & production. Code merged in from https://github.com/department-of-veterans-affairs/vets-website/pull/14099

**Comment by jenstrickland** [comment]
Validated 🎉
